dali/dfu/dfuwu.cpp

```diff
@@ -2437,7 +2499,7 @@ class CDFUWorkUnit: public CDFUWorkUnitBase
         // called with crit locked
         if (!subscriberid) {
             StringBuffer xpath;
-            getXPath(xpath,queryId()).append("/Progress/Edition");
+            getXPath(xpath,queryId()).append("/Progress");///Edition");
```

Comment:
Why this change?  I think it means that changes to Edition won't be notified.

Reply:
Was just me playing with getting the subscribe to work... will revert.  Every other place I look polls for these workunits to complete so I will use the same technique for now.  Perhaps the function that uses subscribe can be fixed in another PR.